### DISCUSSION TITLE
Renderización cauces ríos (ramblas)

### DISCUSSION THREAD

**Discussion by agustincaballero** [manudona92/MapantES#20] [General]
Ahora mismo estamos renderizando todos los ríos de btn con el símbolo 305.0.
Pienso que habría que cambiar para dibujar en 305.0 (continuo) lo que en btn venga como Permanente, y en 306.0 lo que en btn venga como No permanente o como Rambla (el btn distingue esos tres casos).
Al margen de esto, al menos en Canarias, los ríos van "por cualquier sitio", no se adaptan en absoluto a las curvas de nivel.

**Comment by jarufe** [comment]
Estoy de acuerdo. Con la renderización del KP nos teníamos que ceñir a un único símbolo. Al hacerla ad hoc, ya podemos distinguir más casos con los valores de los campos de las tablas del BTN.

**Reply by manudona92** [reply]
OK a todo. 

Idem a lo anterior. Creo que si o si, si no queremos contar con una BTA (que casi todas extraen los cauces de un MDT con más precisión que el BTN25), debemos de asumir los errores geométricos. Al final la curva es la que manda. 

OSM tiene en general (no en todos lados) más densidad de arroyos, pero creo que sigue fallando en geometría como falla la BTN. Lo he hecho a ojo, no con ejemplos claros.

**Reply by manudona92** [reply]
Le he estado dandole otra vuelta, y creo que voy a preguntar a Jesús (es) del IGN si tienen algún SHP que haya extraido del MDT05 (generado con LIDAR) sobre cuencas fluviales. Quizás lo tengan en privado, o la hayan utilizado para modelos de inundabilidad. 

En el centro de descargas del CNIG, en hidrografia, las lineas de cauce son las mismas que las del BTN25, de lo que vi de Lanzarote.

**Comment by jarufe** [comment]
Aunque dije que estaba de acuerdo, hay que ver en qué casos hay que usar símbolos distintos. Al principio de la búsqueda de las masas de agua también vi los diferentes productos que hay relacionados con estudios de inundaciones, etc. Pero no vi que nos fueran a aportar nada mejor. Que una zona tenga probabilidad de inundarse no quiere decir que vaya a tener agua, al menos la mayor parte del año. Y aunque tenga agua, no quiere decir que la masa que se genere sea franqueable o no, que es lo que vienen a señalar los distintos símbolos IOF.

**Comment by manudona92** [comment]
De lo que hablé con Jesús Maria sobre hidrografia, la que está en el centro de descargas (Centro de descargas --> Información Geográfica de Referencia (2º producto) --> Hidrografía.) es la misma que la que está introducida en la BTN25.  Me ha dicho que es la v.0 pero que cree que han estado trabajando en una v.1 generada a través de un MDT más preciso. El responsable de esto vuelve el 12 de enero, ya os aviso de como queda la cosa, en caso de lograr más precisión (de lo que he visto hasta ahora, no está tan tan mal...)

* Respecto a las ramblas, ahí creo que viene también han habido adaptaciones en muchos países. Por ejemplo en climas desérticos o semidesérticos, se ha incluso utilizado el símbolo 304 y 305 en ramblas que solo tienen agua intermitente cada 10-20 años. El grosor del azul indica el ancho del lecho del arroyo/río intermitente. En España, se ha seguido la ISOM más, ya que la mayoria de estas ramblas sin agua se pintan con el 306, que aun así, es un símbolo que ayuda mucho...

Aquí un ejemplo de un mapa de Australia, donde se utiliza 304, 305, 306 para marcar diferentes anchos de cauces. En esa zona llueve quizas 130mm anuales. http://3drerun.worldofo.com/2d/index.php?idmult%5B%5D=172552

Aparte de esto, la ISOM 2017-2 deja claro que la 306 es para aguas no permanentes (intermitentemente) , es decir se incluyen ramblas y aguas no permanentes, como se indica al principio. Las permanentes la dejaría en el 305

![image](https://user-images.githubusercontent.com/76484741/103640892-1ba9f300-4f51-11eb-8507-87a983766bc8.png)

**Reply by jarufe** [reply]
Ok. En mis comentarios estaba pensando más bien en símbolos de tipo área, no en los lineales, para cubrir zonas inundables o zonas que se pueden dibujar a escala por su anchura. El 306 viene muy bien, de hecho, hasta en lugares donde no hay agua ni se le espera, cuando el relieve es muy sinuoso con múltiples vaguadas y espolones y quieres remarcar cuáles son las vaguadas. Así, la composición mental de la tridimensionalidad se vuelve más sencilla.

**Reply by manudona92** [reply]
Entendido. Acabo de ver lo de "masas" de agua ahora. En caso de que haya una categoría de masa "intermitente" de agua, iriamos al símbolo 310 ¿No? En polígonos o masas muy estrechos iria muy limitado este símbolo, es lo único negativo.

**Reply by manudona92** [reply]
Os referís a eso, con lo de las masas de ramblas ? Es un ejemplo de lo recién procesado de Murcia

![image](https://user-images.githubusercontent.com/76484741/103660085-7b15fc00-4f6d-11eb-9d59-90850144fab8.png)

Otro símbolo, a parte del 310 sería este: 
![image](https://user-images.githubusercontent.com/76484741/103660537-fd062500-4f6d-11eb-8063-782e23408e42.png)

**Comment by agustincaballero** [comment]
Disculpad pero en lo de Coto Maravillas no está bien renderizado lo de los ríos.
Una forma de renderizar los ríos permanentes sería así:

![image](https://user-images.githubusercontent.com/18677366/103672187-43fb1700-4f7c-11eb-8b36-21df7be4c82d.png)

Claro, ahí tenemos un problemilla: ese río en cuestión en realidad es normalmente un arroyo de un metro de ancho, totalmente atravesable. Y en este caso además el azul tapa la vegetación de ribera (tarays sobre todo), que a nivel de orientación es muy interesante. 

Pero, naturalmente, en otras regiones los ríos sí suelen tener el ancho que marca el cauce y son menos atravesables.

No sé cuál sería la solución ideal, la verdad.

Sobre ríos no permanentes, yo dejaría la línea central del cauce y quitaría las exteriores.

**Reply by manudona92** [reply]
Sí, me temo que tenemos que aceptar ese tipo de información "engañosa" en aguas permanentes de zonas más secas. Pena porque se pierde información de bosque de ribera...

 Entiendo que son los símbolos siguientes de la MTN25 (no permanente izquierda y permanente derecha ): 
![image](https://user-images.githubusercontent.com/76484741/103676431-5deb2880-4f81-11eb-8ce9-7b43d0cffe0d.png)

OK al símbolo lineal de 406 de ramblas o aguas no permanentes. ¿Por qué? Si utilizaramos símbolo de área, 302 por ejemplo,  perderiamos información valiosa de vegetación y también de curva de nivel. Es muy peculiar este mapa, pero fue liga autonómica en Valencia: https://www.fedocv.org/doma/show_map.php?user=Alex+Tello&map=996
